CesiumGS/cesium
# Updating npm packages

# Description

Just needed `ktx-parse` this time







## Issue number and link

for release





## Testing plan

- Run sandcastle
- Check 3D Models sample for KTX example



# Author checklist
